src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -23,10 +26,69 @@
 
     computed: {
       ...mapState('flow', ['flows']),
+
+      canvasHeight() {
+        const defaultHeight = window.screen.availHeight
+        if (!this.flows[0].blocks && this.flows[0].blocks.length) {
+          return defaultHeight
+        }
+
+        const blockAtTheLowestPosition = lodash.maxBy(this.flows[0].blocks, 'platform_metadata.io_viamo.uiData.yPosition')
```

Comment:
I never realized how backwards this variable is before! `yPosition` is inverted if we're thinking of a cartesian plane, right? Where an upward direction is our position y-value and downward would be a negative or `min()` y-value! 

Did you find that confusing in this task?

Reply:
Ps. Can we swap to `import {maxBy}` from 'lodash' rather than `lodash.maxBy()`?

Reply:
about cartesian plane, I thought about a possible inversion, but I ended with no needed changes. In fact,
In this particular code, we're trying to get Width & Height of the canvas. Meaning, we just need to translate the `yPosition` into a specific Height, and `xPosition` into Width

This is what we get when adding a block:
```
from `platform_metadata.io_viamo.uiData`
{
"xPosition":635,
"yPosition":329
}

related `blockElement`:
clientWidth: 635
clientHeight: 329
```

Therefore, the maxBy is still accurate with positive sign as we look for an absolute value.